### DISCUSSION TITLE
So is this a vue3 type resolution problem?

### DISCUSSION THREAD

**Discussion by LukerSpringtree** [vuejs/devtools#476] [General]
<img width="1349" alt="image" src="https://github.com/vuejs/devtools-next/assets/9697295/d153cded-546b-44d5-aa97-a9173fed2895">

**Comment by LukerSpringtree** [comment]
packages/client/src/components/common/DockingPanel.vue

**Reply by alexzhang1030** [reply]
I'm not sure it's a typescript skill issue or a volar issue
